packages/api-lib/libs/es.js

```diff
@@ -20,7 +20,12 @@ async function connect() {
 
   // use local client
   if (!process.env.ES_HOST) {
-    client = new elasticsearch.Client({ host: 'localhost:9200' })
+    if (!process.env.ES_TEST_HOST) {
+      client = new elasticsearch.Client({ host: 'localhost:9200' })
+    }
+    else {
+      client = new elasticsearch.Client({ host: process.env.ES_TEST_HOST })
+    }
   }
```

Comment:
This is my suggestion to work with a local ES development instance that is not installed in localhost, which is my case.